### ISSUE TITLE
Do not apply relative factor multiple time

### ISSUE THREAD

**Issue by msevestre** [Open-Systems-Pharmacology/OSPSuite.Core#448]
see https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/893

**Comment by msevestre** [comment]
@Yuri05 I don't think this is even possible at the moment. If you have multiple linked parameters using factor, then we still have only one optimized value but MANY values that would to in the simulation.

**Comment by msevestre** [comment]
@Yuri05 Let's have a discussion here. I am not quite happy with the idea of changing so much to prevent user issues clicking too often on a button!

**Comment by msevestre** [comment]
As discussed, not for this version

**Comment by PavelBal** [comment]
https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/156
